### PR TITLE
Fix filename sanitation

### PR DESCRIPTION
**Pull request by de-odex**

### CODE REVIEW

File: slider/library.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def keys(self):
 if sys.platform.startswith('win'):
     def sanitize_filename(name):
         for invalid_character in r':*?"\/|<>[]':
-            name.replace(invalid_character, '')
```
Comment (llllllllll) [inline slider/library.py]: That will do it. One other thing is that we are stripping too many characters, should we remove the ones that are not actually invalid?

Comment (de-odex) [inline slider/library.py]: i'd prefer not to strip characters that aren't the invalid ones

Comment (llllllllll) [inline slider/library.py]: sorry, I totally agree. Some of the characters in the string are in fact valid for windows based on the original screen shot.

**Comment by llllllllll** [comment]
Thank you very much!

**Pull request merged**